handler/processors.go

```diff
@@ -74,11 +80,43 @@ func processPullRequestTargetEvent(e *github.PullRequestTargetEvent, token strin
 	return []int{*e.PullRequest.Number}
 }
 
+func processCronEvent(token string, e *ActionEvent) ([]int, error) {
+	Log("processing 'schedule' event")
+
+	ctx, canc := context.WithTimeout(context.Background(), time.Minute*10)
+	defer canc()
+	client := newGithubClient(ctx, token)
+
+	repoParts := strings.SplitN(*e.Repository, "/", 2)
+	prs, err := GetPullRequests(ctx, client, repoParts[0], repoParts[1])
+	if err != nil {
+		return nil, fmt.Errorf("get pull requests: %w", err)
+	}
+
+	Log("fetched %d prs", len(prs))
+
+	nums := make([]int, 0, len(prs))
+	for _, pr := range prs {
+		nums = append(nums, *pr.Number)
+	}
+
+	Log("found prs %v", nums)
+
+	return nums, nil
+}
+
 // processEvent process the GitHub event and returns the list of pull requests that are affected by the event.
-func ProcessEvent(event *ActionEvent) []int {
+func ProcessEvent(event *ActionEvent) ([]int, error) {
+	// These events do not have an equivalent in the GitHub webhooks, thus
+	// parsing them with github.ParseWebhook would return an error.
```

Comment:
If you have a reference to the official docs with more information, it could be useful to add it in the comment.